README.md

```diff
@@ -11,7 +11,28 @@ The API is contained in `CqlMigrator`, and works on directories of files ending
 
 For example, to apply all `.cql` files located in `/cql` in the classpath:
 
-    CqlMigrator migrator = new CqlMigrator();
+Configure the lock:
+
+    CassandraLockConfig lockConfig = CassandraLockConfig.builder()
+                    .withTimeout(Duration.standardSeconds(3))
+                    .withPollingInterval(Duration.millis(500))
+                    .withClientId("127.0.0.1")
+                    .withSimpleStrategyReplication(3)
+                    .build();
+                    
+Or
+
+    CassandraLockConfig lockConfig = CassandraLockConfig.builder()
+                    .withTimeout(Duration.standardSeconds(3))
+                    .withPollingInterval(Duration.millis(500))
+                    .withClientId("127.0.0.1")
+                    .withNetworkTopologyReplication("DC1", 3)
+                    .withNetworkTopologyReplication("DC2", 3)
+                    .build();
+
+Then:                    
```

Comment:
I feel like we need way simpler config for the locking out of the box. More defaults, for example the client id can default to the hostname for instance. Similarly timeouts can have defaults.